### ISSUE TITLE
nonsyndromic oculocutaneous albinism

### ISSUE THREAD

**Issue by maglott** [monarch-initiative/mondo#641]
For new term requests, please provide the following information:

## Preferred term label

 nonsyndromic oculocutaneous albinism

## Suggested parent term
MONDO_0018910

There is already a term for oculocutaneous albinism.  A submission to ClinVar uses non-syndromic oculocutaneous albinism, which is a term that seems to be used in the literature.  Not sure whether non-syndromic  is the same as what is currently captured by ORPHA:55  or PS203100 (based on the children), but if that is so, non-syndromic should not be indented under oculocutaneous albinism.

Bottom line,  should ORPHA:55  and/or PS203100 be treated as non-syndromic and new node be created for it?

**Comment by nicolevasilevsky** [comment]
I am not an expert on this, perhaps @annieolry or @ahamosh could weigh in?

The abstract for this article makes them suggests these are two different diseases: https://www.sciencedirect.com/science/article/pii/S0022202X15363107

@maglott, do you have a source with a definition for the nonsyndromic term?

**Comment by ahamosh** [comment]
This is regular OCA (which has many features and can certainly be considered a syndrome). OCA6 is not part of a broader syndrome of which albinism is a feature (such as Hermansky-Pudlak syndrome).   Let me know if this needs further clarification.
Thanks,
Ada

Sent from my iPhone

On Mar 11, 2019, at 5:51 PM, Nicole Vasilevsky <notifications@github.com<mailto:notifications@github.com>> wrote:


I am not an expert on this, perhaps @annieolry<https://github.com/annieolry> or @ahamosh<https://github.com/ahamosh> could weigh in?

The abstract for this article makes them suggests these are two different diseases: https://www.sciencedirect.com/science/article/pii/S0022202X15363107

@maglott<https://github.com/maglott>, do you have a source with a definition for the nonsyndromic term?

**Comment by nicolevasilevsky** [comment]
Thanks @ahamosh! I'll add nonsyndromic oculocutaneous albinism as a synonym for MONDO_0018910 'oculocutaneous albinism'

**Comment by maglott** [comment]
But we still have to address the relationships.  Cannot have syndromic indented under a term with a synonym of non-syndromic

**Comment by nicolevasilevsky** [comment]
@maglott I created another PR to address this:
https://github.com/monarch-initiative/mondo/pull/647

@cmungall please let me know if this is the proper approach to reclassify this term

**Comment by nicolevasilevsky** [comment]
The PR was merged, so closing this ticket
